spec/unit/indirector/node/ldap_spec.rb

```diff
@@ -56,15 +59,15 @@
       end
 
       it "should deduplicate class values" do
-        @entry.stubs(:to_hash).returns({})
+        node_indirection.stubs(:ldap_entry_to_hash).returns({})
         node_indirection.stubs(:class_attributes).returns(%w{one two})
         @entry.stubs(:vals).with("one").returns(%w{a b})
         @entry.stubs(:vals).with("two").returns(%w{b c})
         expect(node_indirection.entry2hash(@entry)[:classes]).to eq(%w{a b c})
       end
 
       it "should add the entry's environment to the hash" do
-        @entry.stubs(:to_hash).returns("environment" => %w{production})
+        node_indirection.stubs(:ldap_entry_to_hash).returns(:environment => %w{production})
```

Comment:
Doesn't this mean the ldap terminus is broken when using MRI ruby (as the `ldap_entry_to_hash` method returns symbols as hash keys instead of strings)?